pkgs/applications/science/math/R/default.nix

```diff
@@ -28,12 +28,21 @@ stdenv.mkDerivation (finalAttrs: {
 
   dontUseImakeConfigure = true;
 
-  nativeBuildInputs = [ pkg-config ];
+  nativeBuildInputs = [
+    bison
+    curl
```

Comment:
I'm confused why this needs to be native, I think this is linked against rather than used at build time. I see it's still part of `buildInputs`, but which one will get linked against?

Reply:
I will check and report back.

Reply:
The only reference to curl is in `/nix/store/*-R-4.4.2/lib/R/etc/Makeconf`. Not entirely sure why no library links against curl.

Reply:
Found the other references. Fixed path of curl-config now.